liquidity/liquidity.go

```diff
@@ -563,23 +740,13 @@ func (m *Manager) SuggestSwaps(ctx context.Context, autoloop bool) (
 		return nil, err
 	}
 
-	if summary.totalFees() >= m.params.AutoFeeBudget {
-		log.Debugf("autoloop fee budget: %v exhausted, %v spent on "+
-			"completed swaps, %v reserved for ongoing swaps "+
-			"(upper limit)",
-			m.params.AutoFeeBudget, summary.spentFees,
-			summary.pendingFees)
-
+	err = m.checkSummaryBudget(summary)
```

Comment:
So I mentioned before that easy autoloop's documentation should mention which params are in use. Budget and "max in flight" are two params that are surely used. Maybe could add them to the godocs etc?